product_sticker/models/product_product.py

```diff
@@ -20,12 +20,14 @@ def _get_sticker_arguments(self):
         }
 
     @api.returns("product.sticker")
-    def get_product_stickers(self):
-        """Product Stickers related to this Product Variant and its Template"""
+    def get_product_stickers(self, model_names=None):
```

Comment:
suggestion: if you use compatible data types, reasoning about them becomes simpler. Some suggestions follow the same idea. Here, for example, is easier to work with `()` as a default:

```suggestion
    def get_product_stickers(self, model_names=()):
```